src/View/TwigView.php

```diff
@@ -238,11 +238,12 @@ protected function initializeProfiler(): void
      */
     protected function _render(string $templateFile, array $data = []): string
     {
+        $viewName = Configure::read('TwigView.context.view', '_view');
```

Comment:
Why not use a view option instead of global config?

Reply:
You mean a property? I just forgot to use it.

Reply:
I just implemented it https://github.com/cakephp/twig-view/pull/23 :slightly_smiling_face: